scala-package/macros/src/main/scala/org/apache/mxnet/APIDocGenerator.scala

```diff
@@ -228,21 +228,21 @@ private[mxnet] object APIDocGenerator extends GeneratorBase {
 
     val finalStr =
       s"""/*
-         |* Licensed to the Apache Software Foundation (ASF) under one or more
-         |* contributor license agreements.  See the NOTICE file distributed with
-         |* this work for additional information regarding copyright ownership.
-         |* The ASF licenses this file to You under the Apache License, Version 2.0
-         |* (the "License"); you may not use this file except in compliance with
-         |* the License.  You may obtain a copy of the License at
-         |*
-         |*    http://www.apache.org/licenses/LICENSE-2.0
-         |*
-         |* Unless required by applicable law or agreed to in writing, software
-         |* distributed under the License is distributed on an "AS IS" BASIS,
-         |* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-         |* See the License for the specific language governing permissions and
-         |* limitations under the License.
-         |*/
+         | * Licensed to the Apache Software Foundation (ASF) under one or more
+         | * contributor license agreements.  See the NOTICE file distributed with
+         | * this work for additional information regarding copyright ownership.
+         | * The ASF licenses this file to You under the Apache License, Version 2.0
+         | * (the "License"); you may not use this file except in compliance with
+         | * the License.  You may obtain a copy of the License at
+         | *
+         | *    http://www.apache.org/licenses/LICENSE-2.0
+         | *
+         | * Unless required by applicable law or agreed to in writing, software
+         | * distributed under the License is distributed on an "AS IS" BASIS,
+         | * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+         | * See the License for the specific language governing permissions and
+         | * limitations under the License.
+         | */
```

Comment:
Please not introducing change like this

Reply:
APIDocGenerator.scala is needed, without this change scala style check will failure.